scaffolder-templates/github-workflows/basic-workflow/docs/GithubBasicDoc.md

```diff
@@ -0,0 +1,112 @@
+# Creating a Basic Serverless Workflow via a Software Template with GitHub
+
+## Overview
+This template allows the user to generate a basic workflow project from the user's input.
+The workflow will be generated corresponding to the workflow type (assessment, infrastructure) the user selects.
+
+### basic-workflow as assessment
+An assessment workflow project is generated from the Github Basic Workflow or the Gitlab Basic Workflow templates. This is a simple workflow that evaluates the user's input as string by comparing it to the word `default` to print out the default workflow options or an empty workflow options list.
+Unlike the infrastructure workflow, the output schema of the assessment workflow is crucial to get the workflow options rendered in the UI. In this basic assessment, there is no data output schema validation but the generated workflow provides a sample object for reference which is made of six mandatory fields (currentVersion, upgradeOptions, migrateOptions, newOptions, continuationOptions, otherOptions). See below an example:
+
+```
+"workflowOptions": {
+    "currentVersion": {
+        "id": "DefaultInfrastructureWorkflowTest",
+        "name": "Default infrastructure workflow test"
+    },
+    "upgradeOptions": [],
+    "migrationOptions": [
+        {
+            "id": "MigrationInfrastructureWorkflowTest",
+            "name": "Migration infrastructure workflow test"
+        }
+    ],
+    "newOptions": [],
+    "continuationOptions": [],
+    "otherOptions": []
+}
+```
+
+### basic-workflow as infrastructure
+An infrastructure workflow project is generated from the Github Basic Workflow or the Gitlab Basic Workflow templates. This template prints out the workflow's data input.
+
+## Prerequisites
+
+- An OCP/k8s cluster and an RHDH instance. 
+- A GitHub account
+- An account on Quay.io 
+
+## Steps
+
+To run the template, the user will need to provide input parameters for configuring the serverless workflow.
+
+### Page 1: Workflow Software Project Configuration
+
+On this page the user will provide input for the following parameters:
+
+- Organization Name: The github organization name of the newly created workflow. This Organization needs to exist prior to running the template. 
+- Repository Name: The name of the github repository that will hold the source code for the to-be-created serverless workflow. 
+- Description: Description added to the README file
+- Workflow ID: A unique ID for the serverless workflow in SonataFlow. 
+- Owner: A drop down menu will reveal potential owners for this workflow
+- System: An entity from the catalog.
+
+## Page 2: Workflow Type
+
+On this page, the user will choose a the workflow type (infrastructure / assessment). 
+Upon choosing 'Assesment', the user will be prompted to enter another parameter: 
```

Comment:
Suggestion:
> Upon choosing 'Assesment', the user will be prompted to enter another parameter: 

Upon choosing 'Assessment', the user will be required to provide an additional parameter:

Reply:
And add a new line before to let your text breath
![image](https://github.com/user-attachments/assets/608859db-567f-4e36-b3d3-94d53bc49fc6)